app/src/views/containers/common/hooks/useResetSecondarySidebarCollapse.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { actions } from "store";
+
+export const useResetSecondarySidebarCollapse = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(actions.updateSecondarySidebarCollapse(false));
+  }, [dispatch]);
+};
```

Comment:
There are few more actions that we dispatch throughout the app. for example `product tours`.  Can we make this generic so that we can handle such actions easily?

Reply:
Can you please provide an example for the same?